metadata-ingestion/src/datahub/cli/docker_check.py

```diff
@@ -189,28 +157,32 @@ def to_exception(
 
 def check_docker_quickstart() -> QuickstartStatus:
     container_statuses: List[DockerContainerStatus] = []
-
     with get_docker_client() as client:
         containers = client.containers.list(
             all=True,
             filters=DATAHUB_COMPOSE_PROJECT_FILTER,
         )
+        if len(containers) == 0:
+            return QuickstartStatus([])
+
+        # load the expected containers from the docker-compose file
+        config_file = containers[0].labels.get(
+            "com.docker.compose.project.config_files"
+        )
+        all_containers = []
+        with open(config_file, "r") as config_file:
+            all_containers = yaml.safe_load(config_file).get("services", {}).keys()
```

Comment:
do we know that the file exists? if not, maybe we can fall back to checking all containers in the project

Reply:
it should exists as compose needs a file to start the services. if not, than something deleted it after start.  without the file we have no ways to determine which one belongs to datahub which not. (without using the mapping that we agreed on dumping)

Reply:
ok this should be fine then